ds_judgements_public_ui/sass/includes/_start.scss

```diff
@@ -0,0 +1,12 @@
+.start-form {
```

Comment:
See above comments on _confirmation.scss, the same applies here.

Reply:
@timcowlishaw updated sass file as it is specific _transactional_licence_form_content.scss and BEM 
 block named transactional_licence_form_content.